plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/Xslt.java

```diff
@@ -42,7 +42,9 @@ public static void transformInPlace(IFile file, URL xslt)
     try (InputStream in = file.getContents();
         InputStream stylesheetStream = xslt.openStream();
         InputStream resultStream = applyXslt(in, stylesheetStream)) {
-      file.setContents(resultStream, IFile.FORCE, null /* monitor */);
+        boolean force = true;
+        boolean keepHistory = true;
+        file.setContents(resultStream, force, keepHistory, null /* monitor */);
```

Comment:
Huh, it's a relief that we were actual using Eclipse API, so we had no sync problem.

BTW, indentation is not right.